Library1/Library.fs

```diff
@@ -22,9 +22,389 @@ module FetchData =
 
     [<CompiledName("FetchForecastsAsync")>]
     let fetchForecasts (http:HttpClient) =
-        //http.GetJsonAsync<WeatherForecast[]>("/sample-data/weather.json")
         async {
-            let! forecasts = Async.AwaitTask <| http.GetJsonAsync<WeatherForecast[]>("/sample-data/weather.json")
-            return forecasts
+            return! Async.AwaitTask <| http.GetJsonAsync<WeatherForecast[]>("/sample-data/weather.json")
         }
         |> Async.StartAsTask
+
+namespace BlazorApp1
+
+open System
+open System.Collections.Generic
+open System.Linq
+open System.Threading.Tasks
+open System.Net.Http
+open Microsoft.AspNetCore.Blazor
+open Microsoft.AspNetCore.Blazor.Components
+open Microsoft.AspNetCore.Blazor.Layouts
+open Microsoft.AspNetCore.Blazor.Routing
+
+// NOTE: _ViewImports.cshtml appear to be required in the BlazorApp.
+type _ViewImports () =

```

Comment:
`_ViewImports` appear to be required in the Blazor app. Adding them here did not work. It's possible they would not be necessary in a DSL.